src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -603,13 +610,13 @@ private void commitContractCache(Deposit deposit) {
   }
 
   private void commitStorageCache(Deposit deposit) {
-    storageCache.forEach((key, value) -> {
+    storageCache.forEach((address, storage) -> {
       if (deposit != null) {
         // write to parent cache
-        deposit.putStorage(key, value);
+        deposit.putStorage(address, storage);
       } else {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this "else" keyword to a new dedicated line. ([squid:RightCurlyBraceDifferentLineAsNextBlockCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6pnSgN4uCclo1cm&open=AWbJU6pnSgN4uCclo1cm)

[ISSUE_KEY]: AWbJU6pnSgN4uCclo1cm